PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.webapp/src/com/microsoft/azuretools/webapp/ui/WebAppDeployDialog.java

```diff
@@ -587,6 +594,9 @@ protected IStatus run(IProgressMonitor monitor) {
                 String successMessage = "";
                 String errorMessage = "Error";
                 Map<String, String> postEventProperties = new HashMap<>();
+                postEventProperties.put("runtime",
```

Comment:
why runtime and duration key is lower-cased?  are them special?

Reply:
The data in kusto is all lowercase, so when we collect data just give a lower case name